CMakeLists.txt

```diff
@@ -1048,6 +1048,7 @@ if(WIN32)
   target_compile_definitions(mixxx-lib PUBLIC WINVER=0x0601)
   target_compile_definitions(mixxx-lib PUBLIC _WIN32_WINNT=0x0601)
   if(MSVC)
+    target_compile_definitions(mixxx-lib PUBLIC /UTF8)
```

Comment:
Wait a sec.... shouldn't this be `target_compile_options`? If so, the other PR was also wrong.

Reply:
Oh, you're right...